### PR TITLE
Include the missing tests in the summary about greenwave's decision

### PR DESCRIPTION
**Pull request by pypingou**
This will help making it clearer for user what requirements are currently
missing when one or more tests are missing or have failed.

Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>

### CODE REVIEW

**Comment by pypingou** [comment]
jenkies test

File: bodhi/server/templates/update.html [hunk 1]
```diff
@@ -1044,6 +1044,12 @@ <h3>Related Bugs <span class="badge">${len(update.bugs)}</span></h3>
     <div class="tab-pane" id="automatedtests" role="tabpanel">
       <div id="resultsdb">
       <h3>Automated Test Results</h3>
+      <p>
+        Test results and gating status may some time conflict as the gating status is retrieved
```
Comment (bowlofeggs) [inline bodhi/server/templates/update.html]: s/some time/sometimes/

File: bodhi/server/templates/update.html [hunk 2]
```diff
@@ -1044,6 +1044,12 @@ <h3>Related Bugs <span class="badge">${len(update.bugs)}</span></h3>
...
+        periodically while the test results presented here are retrieved lived. So if your
```
Comment (bowlofeggs) [inline bodhi/server/templates/update.html]: s/periodically/periodically by Bodhi's backend server,/
s/lived/upon page load/
s/So if/If/

**Review by bowlofeggs — APPROVED** [review]
Cool looks nice! Just a few grammatical suggestions.

**Comment by pypingou** [comment]
Grammar fixed, I also added this change: https://github.com/fedora-infra/bodhi/pull/2273/files#diff-7b5829436cd25d51410304eb8e16cefe so the gating status flag in the overview remain concise and don't include the list of missing tests

File: bodhi/server/util.py [hunk 3]
```diff
@@ -549,7 +549,8 @@ def test_gating_status2html(context, status, reason=None):
     html = u'<span class="label label-%s">Tests %s</span>' % (label_class, description)
     from bodhi.server.models import TestGatingStatus
     if status not in [None, TestGatingStatus.ignored, TestGatingStatus.passed] and reason:
-        html += u'<p><a class="gating-summary" href="#automatedtests">%s</a></p>' % reason
+        html += u'<p><a class="gating-summary" href="#automatedtests">%s</a></p>' % (
+                reason.split('\n')[0])
```
Comment (pypingou) [inline bodhi/server/util.py]: split('\n', 1)

**Review by bowlofeggs — APPROVED** [review]

**Pull request merged**